android/src/main/java/io/mosip/tuvali/ble/peripheral/Peripheral.kt

```diff
@@ -36,7 +38,13 @@ class Peripheral(context: Context, peripheralListener: IPeripheralListener) {
   }
 
   private fun notifyDisconnect(serviceUUID: UUID) {
-    sendData(serviceUUID, GattService.DISCONNECT_CHAR_UUID, byteArrayOf(Verifier.DISCONNECT_STATUS.toByte()))
+    val data = byteArrayOf(Verifier.DISCONNECT_STATUS.toByte())
```

Comment:
I think we should refactor on develop. Peripheral layer being the lower layer should avoid having business logic. It should just send data to whatever characteristic given by higher layers.

At the same time, adding checksum data can be encapsulated into DisconnectCharData class.